### PR TITLE
Feat: upgrade dependencies: axum to 0.7 and leptos 0.6. Fix axum- and actix-examples

### PR DESCRIPTION
**Pull request by LouneCode**
# Upgrade dependencies: axum to 0.7 and leptos 0.6

PR upgrades leptos_sse  to use the lates leptos and axum versions. Also both axum and actix-examples are fixed.

Fixed examples assume you’re using nightly version of Rust and the nightly feature of Leptos. To use nightly Rust, you can set your toolchain on per-project basics by adding rust-toolchain.toml file in to the project.

rust-toolchain.toml file is added in both example folders.  
```
[toolchain]
channel = "nightly"
targets = ["wasm32-unknown-unknown"]
```

## Upgrade background information and ideas

- [Leptos discussions - 0.6.0-rc1  ](https://github.com/leptos-rs/leptos/discussions/2226)
- [headless-studio/leptos-leaflet - Update to leptos 0.6.x](https://github.com/headless-studio/leptos-leaflet/commit/c964408aff456fe5ff915123f1a113a0af84c9ff)
- feat: upgrade axum to 0.7.4 #7

<BR />
<BR />
<BR />

**Let's do IT better**

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
```
Comment (messense) [inline rust-toolchain.toml:2]: Please don't add this in project root, adding it in examples are fine.

**Comment by LouneCode** [comment]
OK, I'll remove the rust-toolchain.toml file from the root of the project and see if the project still compiles.

**Comment by LouneCode** [comment]
It seems Axum 0.7 depends on leptos 0.6 and this project won't compile without the "night" channel toolchain setting at the moment. This also applies to the actix example, as it refers to the sse_events library, which should be the same version level and use the same leptos version 0.6.

Maybe this change should postpone until the project compiles without "nightly" channel configuration?

Do you have any ideas how to solve this problem?

**Comment by LouneCode** [comment]
rust-toolchain.toml removed from project root folder as requested. 
I was able to compile this PR without errors after running the rustup update
```rustup update``` 
<BR />
Compile comman in root folder:  
```cargo build```

**Review by messense — APPROVED** [review]

**Pull request merged**